fake.py

```diff
@@ -297,10 +297,26 @@ def compose_frame(self, frame):
         if self.no_background is False:
             background_frame = next(self.images["background"])
         else:
-            background_frame = cv2.blur(frame,
-                                        (self.background_blur,
-                                         self.background_blur),
-                                        cv2.BORDER_DEFAULT)
+
+            # copy frame by downscaling 
+            frameWidth = frame.shape[1]
+            frameHeight = frame.shape[0]
+
+            width = int(frameWidth * .25)
+            height = int(frameHeight * .25)
```

Comment:
These two lines - give them a shot at `.5` or `.75`, see if it looks more natural on your camera

Reply:
Hmm it doesn't really matter. I don't really use blurring myself. The whole point is to make the fine features of the background unrecognisable. Having 0.25 as a parameter achieves that goal and performance.